pandas-stubs/core/indexes/multi.pyi

```diff
@@ -27,31 +29,42 @@ from pandas._typing import (
 class MultiIndex(Index[Any]):
     def __new__(
         cls,
-        levels=...,
-        codes=...,
-        sortorder=...,
+        levels: SequenceNotStr[SequenceNotStr[Hashable]] = ...,
+        codes: SequenceNotStr[SequenceNotStr[int]] = ...,
+        sortorder: int | None = ...,
         names: SequenceNotStr[Hashable] = ...,
-        dtype=...,
-        copy=...,
+        copy: bool = ...,
         name: SequenceNotStr[Hashable] = ...,
         verify_integrity: bool = ...,
         _set_identity: bool = ...,
     ) -> Self: ...
     @classmethod
     def from_arrays(
-        cls, arrays, sortorder=..., names: SequenceNotStr[Hashable] = ...
+        cls,
+        arrays: SequenceNotStr[SequenceNotStr[Hashable]] | SequenceNotStr[AnyArrayLike],
```

Comment:
This one's a bit odd

The name is `from_arrays`, and the pandas docstring says that this should be a sequence of array-like

But, the pandas docs (and tests) have several examples of passing lists, which are not `ArrayLike`

`lists` are not included in `AnyArrayLike`, so I've used `SequenceNotStr[Hashable] | AnyArrayLike`